scripts/check-bundle-dependencies.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+function failMessage() {
+    echo -e "\033[0;31mTest with  values=\"$1\" and expected_charts=\"$1\" Failed\033[0m"
+}
+
+function checkDependencies()
+{
+    local values="$1"
+    local expected_charts="$2"
+    [[ "$values" == "" ]] && local values_set="global.cluster=test,global.licenseKey=fake" || local values_set="global.cluster=test,global.licenseKey=fake,$values"
+    chart_sources=$(mktemp)
+    helm template charts/nri-bundle --set $values_set | grep "# Source" > $chart_sources
+    # Check no unexpected chart would be installed
+    local expected_charts_re=${expected_charts// /|} # replace ' ' with '|' "x y" -> "x|y"
```

Comment:
If you don't use quotes, multiple spaces become only one space.
```suggestion
    local expected_charts_re="${expected_charts// /|}" # replace ' ' with '|' "x y" -> "x|y"
```